Cirrious.FluentLayout/Margins.cs

```diff
@@ -40,7 +40,15 @@ public Margins(float allHorizontal, float allVertical)
             HSpacing = allHorizontal;
         }
 
-        public Margins(float left, float top, float right, float bottom, float hspacing, float vspacing)
+		public Margins(float left, float top, float right, float bottom)
+		{
+			Top = top;
+			Bottom = bottom;
+			Right = right;
+			Left = left;
+		}
+
+		public Margins(float left, float top, float right, float bottom, float hspacing, float vspacing)
```

Comment:
Rather than add yet another constructor here I think it would be nicer to just make the parameters here all have defaults of `0`, allowing you to just set the ones you want